frontend/src/components/Dropdown/Dropdown.styled.jsx

```diff
@@ -0,0 +1,95 @@
+import styled from "styled-components";
+import {breakpoints} from "../../rwd/breakpoints";
+
+export const StyledDropdown = styled.div`
```

Comment:
Nie musimy używać przedrostka Styled.

Reply:
W tym przypadku tak, jest const Dropdown, gdyby było bez Styled, wywalałoby błąd "Dropdown was already declared".

Reply:
nie będzie already declared jeżeli używasz tych stylów do pliku w którym nie ma Dropdown
możliwe, że jak robiłeś import {dropdown} to zaciągnęło Ci style z innego pliku.styled.js w którym już deklarowałeś dropdown